packages/util/test/account.spec.ts

```diff
@@ -810,3 +815,342 @@ describe('Utility Functions', () => {
     assert.equal(JSON.stringify(result), JSON.stringify(RLP.encode(body)))
   })
 })
+
+describe('createPartialAccount', () => {
+  it('should throw an error when all fields are null', () => {
+    assert.throws(
+      () =>
+        createPartialAccount({
+          nonce: null,
+          balance: null,
+          storageRoot: null,
+          codeHash: null,
+          codeSize: null,
+          version: null,
+        }),
+      'All partial fields null',
+    )
+  })
+
+  it('should return Account with correct values when fields are provided', () => {
+    const account = createPartialAccount({
+      nonce: 1n,
+      balance: Units.ether(1),
+      storageRoot: KECCAK256_RLP,
+      codeHash: KECCAK256_RLP,
+      codeSize: 10,
+      version: 1,
+    })
+
+    assert.deepEqual(account, new Account(1n, Units.ether(1), KECCAK256_RLP, KECCAK256_RLP, 10, 1))
+  })
+
+  it('should return Account with null and undefined value fields as given', () => {
+    const account = createPartialAccount({
+      nonce: undefined,
+      balance: Units.ether(1),
+      storageRoot: undefined,
+      codeHash: null,
+      codeSize: 10,
+      version: undefined,
+    })
+
+    assert.deepEqual(
+      account,
+      new Account(undefined, Units.ether(1), undefined, null, 10, undefined),
+    )
```

Comment:
The same for this test, use a `const` data variable to read from.

Reply:
Fixed